unlock-app/src/components/interface/locks/Create/elements/CreateLockForm.tsx

```diff
@@ -60,15 +60,22 @@ export const networkDescription = (network: number) => {
           Mainnet.
         </p>
       )}
-      {faucet && (
+      {faucets && (
         <>
           {' '}
           <br />
-          Need some {nativeCurrency.name} to pay for gas?{' '}
-          <Link className="underline" href={faucet} target="_blank">
-            Try this faucet
-          </Link>
-          .
+          Need some {nativeCurrency.name} to pay for gas? Try one of these
+          faucets:{' '}
+          {faucets.map((faucet: any) => {
```

Comment:
Please handle the case where there is just 1 faucet distinctly so it is not awkward to say "try these faucets" and there is just one.